toolchain/check/impl.cpp

```diff
@@ -163,7 +163,7 @@ static auto BuildInterfaceWitness(
           CARBON_FATAL() << "Unexpected type: " << type_inst;
         }
         auto& fn = context.functions().Get(fn_type->function_id);
-        auto impl_decl_id = context.LookupNameInExactScope(
+        auto [impl_decl_id, access_kind] = context.LookupNameInExactScope(
```

Comment:
```suggestion
        auto [impl_decl_id, _] = context.LookupNameInExactScope(
```

Idiomatically, unused fields are given the `_` name to indicate they're discarded.